oak-blob-cloud/src/main/java/org/apache/jackrabbit/oak/blob/cloud/s3/S3Backend.java

```diff
@@ -421,15 +414,20 @@ public InputStream read(DataIdentifier identifier)
         ClassLoader contextClassLoader = Thread.currentThread().getContextClassLoader();
         try {
             Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
-            S3Object object = s3service.getObject(bucket, key);
-            InputStream in = object.getObjectContent();
-            LOG.debug("[{}] read took [{}]ms", identifier, (System.currentTimeMillis() - start));
+            InputStream in = s3Client.getObject(s3ReqDecorator.decorate(
+                    GetObjectRequest.builder()
+                            .bucket(bucket)
+                            .key(key)
+                            .build()),
+                    ResponseTransformer.toInputStream());
+
+            LOG.debug("[{}] read took [{}]ms", identifier, System.currentTimeMillis() - start);
```

Comment:
Stopwatch?

Reply:
They were written using `System.currentTimeMillis`, so we used that to keep the symmetry.

Reply:
you can get millis from StopWatch as well. but yes, just a nit